qkernel/src/interrupt/aarch64/mod.rs

```diff
@@ -331,77 +336,49 @@ pub fn GetFaultAccessType(esr:u64, is_exe:bool) -> AccessType{
 
 }
 
-pub fn MemAbortUser(ptregs_addr:usize, esr:u64, far:u64, is_instr:bool){
-    debug!("get {} abort fault from el0",
-           match is_instr {
-               true  => "instruction",
-               false => "data",
-           });
-    let dfsc = esr & EsrDefs::ISS_DFSC_MASK;
-    let access_type = GetFaultAccessType(esr, is_instr);
-    let dfsc_root = dfsc & PageFaultErrorCode::GEN_xxSC_MASK;
-    match dfsc_root {
-        PageFaultErrorCode::GEN_PERMISSION_FAULT |
-        PageFaultErrorCode::GEN_TRANSLATION_FAULT|
-        PageFaultErrorCode::GEN_ACCESS_FLAG_FAULT => {
-            info!("DFSC/IFSC == {:#X}, FAR == {:#X}, acces-type fault == {}, \
-                  during address translation == {}.", dfsc, far,
-                  access_type.String(), EsrDefs::IsCM(esr));
-            let ptregs_ptr = ptregs_addr as *mut PtRegs;
-            let ptregs = unsafe {
-                &mut *ptregs_ptr
-            };
-            //
-            //TODO: on work
-            //
-            let error_code = PageFaultErrorCode::new(true, esr);
-            PageFaultHandler(ptregs , far, error_code);
-            return;
+
+pub fn HandleMemAbort(ptregs_addr: usize, esr: u64, far: u64, is_instr: bool, is_user: bool) {
+    debug!(
+        "get {} abort fault from {}, esr: {:x}",
```

Comment:
I prefer hex values to be formatted with '{:#x}', as it avoids any confusion.

Reply:
I'll fix in later commits.